homeassistant/components/rainmachine/switch.py

```diff
@@ -497,6 +510,16 @@ def update_from_latest_data(self) -> None:
                     data["waterSense"]["precipitationRate"], 2
                 )
 
+        if self._entry.options[CONF_USE_APP_RUN_TIMES]:
+            if (
```

Comment:
Although correct, the way `black` styles this conditional is ugly. Can we save the provision settings in a variable, then access the zone duration from that variable (to avoid a bunch of indention)?

Reply:
Updated.